requirements/edx-sandbox/base.txt

```diff
@@ -11,22 +11,24 @@ common/lib/symmath
 asn1crypto==0.24.0
 backports-abc==0.5        # via tornado
 cffi==1.11.5
-cryptography==2.2.2
+cryptography==2.4.2
```

Comment:
I think these changes have to be kept very narrow.  This is pulling in other updates.